lib/utils/templatize.html

```diff
@@ -165,6 +165,18 @@
               } else {
                 n.textContent = n.__polymerTextContent__;
               }
+            // remove and replace slot
+            } else if (n.localName === 'slot') {
+              if (hide) {
+                n.__polymerParent__ = n.parentNode;
+                n.__polymerSibling__ = n.nextSibling;
```

Comment:
If the sibling was removed (e.g. due to `dom-if` going false) then the slot would never come back.

May be better to replaceChild with a non-slot?